Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Climate Modeling Alliance"]
 version = "0.4.0"
 
 [deps]
+ArtifactWrappers = "a14bc488-3040-4b00-9dc1-f6467924858a"
```

Comment:
This file should not have been edited for this PR, and we shouldn't be adding ArtifactWrappers to the Project.toml